docs/Preparation.md

```diff
@@ -0,0 +1,46 @@
+# Environment Preparation
+
+This document describes how to ensure your environment is configured
+for working with Moodle on Azure.
+
+## Prerequisites
+
+In order to configure our deployment and tools we'll set up some
+[environment variables](./Environment-Variables.md) to ensure consistency.
+
+## Required software
+
+We'll use a number of tools when working with Moodle on Azure. Let's
+ensure they are all installed:
+
+``` shell
+sudo apt-get update
+sudo apt-get install wget -y
+sudo apt-get openssh-client -y
+```
```

Comment:
We don't need Azure CLI 2.0, or is it installed somewhere else?

Reply:
Done.